moduleroot/.travis.yml.erb

```
@@ -129,6 +129,9 @@ jobs:
       - "bundle exec rake 'litmus:install_agent[<%=puppet_version%>]'"
       - "bundle exec rake litmus:install_module"
       bundler_args:
+<%  if config['collection']['dist']-%>
+      dist: <%=config['collection']['dist']%>
+<% end -%>
```

Slight nitpick for alignment:
```suggestion
<%  end -%>
```